server/README.md

```diff
@@ -11,6 +11,25 @@
 
 3. Run the application with: `sbt run`
 
+## Troubleshooting
+
+1. If when you run xsnd you get 
```

Comment:
If you get the following error while running the explorer: `error: Could not locate RPC credentials.`, then, create a file at`~/.xsncore/xsn.conf` with this content:
    ```
    rpcuser=dummy
    rpcpassword=replaceme
    rpcport=51473
    txindex=1
    ```

Now, `xsnd` can be started without arguments, for example, `./bin/xsnd`.